label_maker/main.py

```diff
@@ -85,12 +85,11 @@ def cli():
 
     # custom validation for top level keys
     # require either: country & bounding_box or geojson
-    if 'geojson' not in config.keys() and not ('country' in config.keys() and 'bounding_box' in config.keys()):
-        raise Exception('either "geojson" or "country" and "bounding_box" must be present in the configuration JSON')
+    if not ('country' in config.keys() and 'geojson' in config.keys()) and not ('country' in config.keys() and 'bounding_box' in config.keys()):
```

Comment:
This change relates to the change of the `filtered_geo` variable, where I interpreted the `geojson` attribute to be an alternative to provide the AOI  instead of `country+bbox` combination.